source/common/protobuf/utility.cc

```diff
@@ -196,14 +197,9 @@ void deprecatedFieldHelper(Runtime::Loader* runtime, bool proto_annotated_as_dep
   std::string with_overridden = fmt::format(
       error,
       (runtime_overridden ? "runtime overrides to continue using now fatal-by-default " : ""));
-  if (warn_only) {
```

Comment:
So this code part is now in `onDeprecatedField` method in strict & warning visitors @htuch . Worth to note that new exception type `DeprecatedFieldException` has been introduced instead of `ProtoValidationException`.